src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -108,6 +108,15 @@ export const ExecutiveFirstPage = ({ data }) => {
 
         <Text style={styles.execHeading}>Breakdown of registered systems</Text>
 
+        {
+            nonPSICount > 0
```

Comment:
Shouldn’t this also include `psiEnabledCount`? Along with `totalCount `
```suggestion
            nonPSICount > 0 && psiEnabledCount > 0
```

Reply:
Hello @r14chandra, 
Here we are considering only `nonPSICount` and not `totalCount`. We need to show the message only when there are nonPSIEnabled systems available and hence I had added the above condition. You can go through the test scenarios in the card description which I had discussed with Pau and Stefan and let me know if you have any further queries! For now marking it resolved, feel free to open the comment if needed 💯